src/main/java/uk/gov/justice/hmpps/prison/service/transfer/TrustAccountService.kt

```diff
@@ -4,26 +4,31 @@ import org.slf4j.LoggerFactory
 import org.springframework.context.annotation.Profile
 import org.springframework.stereotype.Service
 import uk.gov.justice.hmpps.prison.repository.FinanceRepository
+import uk.gov.justice.hmpps.prison.repository.jpa.model.AgencyLocation
 import uk.gov.justice.hmpps.prison.repository.jpa.model.ExternalMovement
 import uk.gov.justice.hmpps.prison.repository.jpa.model.OffenderBooking
 
 interface TrustAccountService {
-  fun createTrustAccount(booking: OffenderBooking, movementOut: ExternalMovement, movementIn: ExternalMovement)
+  fun createTrustAccount(
+    booking: OffenderBooking,
+    fromAgency: AgencyLocation,
```

Comment:
pretty certain we don't need to pass in fromAgency since it is in the movement, but that will be a later PR